### ISSUE TITLE
Add -webkit-font-smoothing: antialiased

### ISSUE THREAD

**Issue by mustafa0x** [quran/quran.com-frontend#990]
Makes the font look better on Mac.

```css
.word {
  -webkit-font-smoothing: antialiased;
}
```

**Comment by aliasaria** [comment]
Hate to be pedantic but I think this one is debatable. Safari default anti-aliasing is:

`-webkit-font-smoothing: subpixel-antialiased;
`

And _officially_, this is the most _accurate_ form of anti-aliasing for modern monitors.

A bit more is written up on this here:

http://usabilitypost.com/2010/08/26/font-smoothing/

I agree that the fonts could be a bit less heavy but I suggest that the solution is fix the font, rather than overriding safari's default, more detailed anti-aliasing model.

I suggest we leave as is.

**Comment by mustafa0x** [comment]
Yeah, these old fonts lose accuracy (e.g. the "holes" in a lot of glyphs become filled or nearly filled) with the better form of font smoothing. That's why quran.com uses antialiased for the surah names on the front page. The newer QPC fonts don't have this issue.

I compared with Windows, and it's smoothing of these fonts is comparable to OSX's antialiased 

I also attached some screenshots for comparison.

**Default smoothing**
![font-smoothing-default](https://user-images.githubusercontent.com/1473102/44029082-ba0716ae-9f04-11e8-8838-9c63b716da55.png)

**Antialiased**
![font-smoothing-antialiased](https://user-images.githubusercontent.com/1473102/44029083-ba2cfb08-9f04-11e8-8fe6-d384ecbba03d.png)
